ci/github/coverity.sh

```diff
@@ -0,0 +1,53 @@
+#! /bin/bash
+#
+# Copyright 2017 - 2022 James E. King III
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE_1_0.txt or copy at
+#      http://www.boost.org/LICENSE_1_0.txt)
+#
+# Bash script to run in GHA to perform a Coverity Scan build
+# To skip the coverity integration download (which is huge) if
+# you already have it from a previous run, add --skipdownload
+#
+
+#
+# Environment Variables (you must define)
+#
+# COVERITY_SCAN_NOTIFICATION_EMAIL  - email address to notify
+# COVERITY_SCAN_TOKEN               - the Coverity Scan token (should be secure)
+#
+
+set -ex
+
+. "$(dirname "${BASH_SOURCE[0]}")"/../enforce.sh
+: ${B2_TARGETS:="libs/$SELF/test"}
+: ${CLANG_COMPILER:="$(echo ${B2_COMPILER} | cut -d'-' -f1)++-$(echo ${B2_COMPILER} | cut -d'-' -f2)"}
+
+sudo wget -nv https://entrust.com/root-certificates/entrust_l1k.cer -O /tmp/scanca.cer
+
+pushd /tmp
+if [[ "$1" != "--skipdownload" ]]; then
+  rm -rf coverity_tool.tgz cov-analysis*
+  curl --cacert /tmp/scanca.cer -L -d "token=${COVERITY_SCAN_TOKEN}&project=${GITHUB_REPOSITORY}" -X POST https://scan.coverity.com/download/cxx/linux64 -o coverity_tool.tgz
+  tar xzf coverity_tool.tgz
+fi
+COVBIN=$(echo $(pwd)/cov-analysis*/bin)
+export PATH=${COVBIN}:${PATH}
+popd
+
+RESULTS_DIR="$(pwd)/cov-int"
+mkdir -p "${RESULTS_DIR}"
+# for re-running manually
+rm -rf "${RESULTS_DIR}/*"
+cov-configure --template --compiler "${CLANG_COMPILER}" --comptype "clangcxx"
```

Comment:
```suggestion
cov-configure --template --compiler "$CXX" --comptype "clangcxx"
```

Maybe check if this is really clang, i.e. check if `B2_TOOLSET == 'clang'` or `if [[ "$CXX" == clang* ]]`

Reply:
Ok.